letsencrypt/domains.sls

```diff
@@ -3,8 +3,17 @@
 
 {% from "letsencrypt/map.jinja" import letsencrypt with context %}
 
+{% if letsencrypt.use_package %}
```

Comment:
May be that we should ensure that the two scripts in _/usr/local/bin_  are _file.absent_ in this case ?

Reply:
@aanriot good point, otherwise this formula leaves files behind which aren't managed.

Reply:
That's a good point. I'll add these changes. Also, do we want to remove the whole **/opt/letsencrypt** dir when using pakages, or do we leave it there, JIC the user of the formula left something important? Personally, I think the latter is the best path, I can add a warning in the readme/pillar about this. Wdyt?

Reply:
@javierbertoli leaving the directory as is and set a warning in README is a safe option,